jenkins-agents/jenkins-agent-helm/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM quay.io/openshift/origin-jenkins-agent-base:4.6
 
 ARG VERSION=3.2.0
-ARG YQ_VERSION=3.3.0
+ARG YQ_VERSION=4.5.1
```

Comment:
@Tompage1994 ; the version needs the `v` added, as per your other PR fix due to html.

Reply:
Just keeping you on your toes...

Fixed now